### DISCUSSION TITLE
Wizard is triggered by each input, what am I doing wrong?

### DISCUSSION THREAD

**Discussion by bouchtaoui-dev** [telegraf/telegraf#1924] [Q&A] [answered]
Hello friends,

First I want to thank all the volunteers who made this library possible 👏🏼

I managed to create a Wizard and it works ok. But the problem is, it keeps restarting the Wizard again when I receive a new message (a command, text or whatever). The code is below on the link.

[Telegraf code example](https://gist.github.com/bouchtaoui-dev/38b74f035e432819aac183500ee7f8f1)

I'll improve the code later, but first I want to get it working in the most basic way.

Maybe you guys can tell me what I'm doing wrong that keeps my Wizard triggered every time.
I hope you can help me with this, thank you.

**Comment by bouchtaoui-dev** [comment, marked as answer]
Well, I finally found the problem:

```js
const stage = new Stage([questionWizard], { default: "create-question" });
```
This default option causes to automatically run the scene every time a message is received.

The solution was simply:
```js
const stage = new Stage([questionWizard], {});
```
That's the disadvantage of copying code :/

**Reply by MKRhere** [reply]
Maybe we should just remove it in the example since it's not what most users want. Thanks for the feedback!
